core/modules/widgets/keyboard.js

```diff
@@ -88,7 +89,8 @@ Selectively refreshes the widget if needed. Returns true if the widget or any of
 */
 KeyboardWidget.prototype.refresh = function(changedTiddlers) {
 	var changedAttributes = this.computeAttributes();
-	if(changedAttributes.message || changedAttributes.param || changedAttributes.key || changedAttributes["class"] || changedAttributes.tag) {
+	if(changedAttributes.message || changedAttributes.param || changedAttributes.key || changedAttributes["class"] || changedAttributes.tag ||
+	  (this.shortcutKeyDescriptor && changedTiddlers["$:/config/shortcuts/" + this.shortcutKeyDescriptor])) {
```

Comment:
hmmm. Is it needed to hardcode the `$:/config/shortcuts` tiddlers ?? ... When ever the store is changed, the refresh should be triggered automatically. ...

For me this seems to be "fighting the symptoms" of an issue and not fixing the issue!!

Reply:
There are the different namespaces for the different operating systems... I've realized that now, therefor I need to change it... I can check for the os like it's done in parseKeyDescriptors()

In $:/core/modules/keyboard.js it's already sort of hardcoded, but it considers the different OS